db/seeds.rb

```diff
@@ -140,14 +146,14 @@
 end
 
 # Add patients for reporting purposes - CSV exports, fulfillments, etc.
+PaperTrail.request.whodunnit = user.id
```

Comment:
is this setting it for the scope of the rest of the file?

Reply:
correct, yeah - if there isn't a block executing where we temporarily set whodunnit, this will be the fallback for everything else.